R/msgmerge.R

```diff
@@ -70,3 +79,66 @@ update_en_quot_mo_files <- function(dir, verbose) {
   }
   return(invisible())
 }
+
+
+
+#' Create or update a `.po` file containing translations
+#'
+#' @description
+#' * `po_create()` creates a new `po/{lang}.po` containing the messages to be
+#'   translated.
+#' * `po_update()` updates an existing `.po` file after the messages in a
+#'   package have changed. The translations for existing messages are preserved;
+#'   new messages are added; and translations for deleted message are marked
+#'   as deprecated and moved to the bottom of the file.
+#'
+#' @param lang Language identifiers. These are typically two letters (e.g.
```

Comment:
This basically repeats the description in `?translate_package`... `@inheritParams` doesn't feel quite right because of how detailed we want to be... Maybe we need a `?language-code` .Rd file we can refer to? `@param languages Language identifiers. See [language-code] for details.`?